src/Frontend/Themes/Bootstrap/src/Js/index.js

```diff
@@ -4,7 +4,7 @@ import Bootstrap from 'bootstrap-sass/assets/javascripts/bootstrap';
 /* Utilities imports */
 import SweetScroll from "sweet-scroll";
 import resizeFunction from './utilities/resize';
-/* eg. import fancyBox from './utilities/fancybox'; */
+/* eg. import {fancyBox} from './utilities/fancybox'; */
```

Comment:
As discussed: I would recommend to also add the Utilities/Fancybox.js file which import the plugin and just initializes Fancybox.